### DISCUSSION TITLE
stplanr 1.0.0

### DISCUSSION THREAD

**Discussion by Robinlovelace** [ropensci/stplanr#489] [General]
- Remove dependency on `sp`, `rgeos` and `rgdal` (#332)
- That involved removal of the following functions:
  - [catchmentArea.R](https://github.com/ropensci/stplanr/blob/v0.8.5/R/catchmentArea.R)
  - Some of the functionality from [linefuns.R](https://github.com/ropensci/stplanr/blob/v0.8.5/R/linefuns.R)
  - Browse the code base as of stplanr 0.8.5 here: https://github.com/ropensci/stplanr/blob/v0.8.5
- Removal of 'ABS' reading functionality in favour of https://github.com/mattcowgill/readabs

<hr /><em>This discussion was created from the release <a href='https://github.com/ropensci/stplanr/releases/tag/v1.0.0'>stplanr 1.0.0</a>.</em>

**Comment by stevenpandrews** [comment]
@Robinlovelace Do you happen to know of good alternative packages for the catchment area style of analysis?

**Reply by Robinlovelace** [reply]
Not sure, @rafaelpeira may have some ideas on the R side..

**Reply by Robinlovelace** [reply]
cc @rafapereirabr

**Reply by stevenpandrews** [reply]
In case anyone else finds this useful, we're looking at `spnetworks`, `sfnetworks`, `r5r`, `openrouteservice`, `dodgr`, and `mapboxapi` which all have some flavor of isochrone functionality.

**Reply by rafapereirabr** [reply]
Hi @stevenpandrews. I think you've already found some of the key libraries. I list a few others in the 3rd page of my slides [here](https://ipeagit.github.io/access_workshop_bogota_2024/2.1_installation.html) and on this [post](https://www.urbandemographics.org/post/open-source-transport-network-analysis/).
@Robinlovelace , thanks for the ping !

**Reply by Robinlovelace** [reply]
Cool, let us know how it goes. If your explorations go well and you do manage to get catchment area analysis going with open source software for reproducible research, feel free to ping me test it out and provide feedback. Definite need for more in this space so good luck!
